src/wrapper-start.js

```diff
@@ -37,20 +37,13 @@
     var newErr;
     if (err instanceof Error) {
       var newErr = new Error(err.message, err.fileName, err.lineNumber);
-      if (isBrowser) {
-        newErr.message = err.message + '\n\t' + msg;
-        newErr.stack = err.stack;
-      }
-      else {
-        // node errors only look correct with the stack modified
-        newErr.message = err.message;
-        newErr.stack = err.stack + '\n\t' + msg;
-      }
+      newErr.message = err.message + '\n\t' + msg;
+      newErr.stack = err.stack;
     }
     else {
-      newErr = err + '\n\t' + msg;
+      newErr = msg + '\n\t' + err;
```

Comment:
I think this order still needs to be reversed again to match the same semantics?